javascript/packages/orchestrator/src/providers/k8s/kubeClient.ts

```diff
@@ -543,10 +543,7 @@ export class KubeClient extends Client {
       },
       {
         type: "deployment",
-        files: [
-          settings.backchannel ? "backchannel-pod.yaml" : null,
-          "fileserver-pod.yaml",
-        ],
+        files: [settings.backchannel ? "backchannel-pod.yaml" : null],
```

Comment:
```suggestion
        files: settings.backchannel ? ["backchannel-pod.yaml"] : [],
```

Reply:
Thanks, I didn't saw this before merge. Can you add it in a follow up pr?
Thx!